jsonnet/node-exporter.libsonnet

```diff
@@ -138,7 +138,19 @@ function(params)
                       // gather that data (especially for bare metal clusters), and
                       // add flags to collect the node_cpu_info metric + metrics
                       // from the text file.
-                      args: [a for a in c.args if a != '--no-collector.hwmon'] + ['--collector.cpu.info', '--collector.textfile.directory=' + textfileDir],
+                      args: [a for a in c.args if a != '--no-collector.hwmon'] +
+                            [
+                              '--collector.cpu.info',
+                              '--collector.textfile.directory=' + textfileDir,
+
+                              // The cpufreq collector seems to be causing high
+                              // load on some nodes with lots of cores.  Disable
+                              // it temporarily as a workaround.
+                              //
+                              // https://bugzilla.redhat.com/show_bug.cgi?id=1972076
+                              // https://github.com/prometheus/node_exporter/issues/1880
```

Comment:
Why do we want to do this? It does not seem related to the bug reported

Reply:
correct, that shouldnt be there :)